tests/test_data_utils.py

```diff
@@ -195,6 +195,33 @@ def test_maybe_apply_chat_template(self, tokenizer_id, example):
             self.assertIsInstance(result["label"], bool)
             self.assertEqual(result["label"], example["label"])
 
+    def test_apply_chat_template_with_tools(self):
+        tokenizer = AutoProcessor.from_pretrained("NousResearch/Hermes-2-Pro-Llama-3-8B")
```

Comment:
Can we replace this by any of our [tiny models](https://huggingface.co/collections/trl-internal-testing/tiny-models-674b1f53bd1085e7d3c20fde)?